### ISSUE TITLE
Groups shouldn't have their own timelines

### ISSUE THREAD

**Issue by zrispo** [zrispo/wick-editor#806]

**Comment by kryptot7** [comment]
I agree that groups shouldn't have frames, because that could be confusing and you'd be better off using a clip. However, I like the fact that groups have their own layers. It makes them good for organization, so you can contain one-time-use shapes without cluttering your asset library (assuming that clips and buttons will show up in the asset library at some point).
